src/executor/graph_executor.cc

```diff
@@ -1581,7 +1583,7 @@ GraphExecutor::CachedSegOpr GraphExecutor::CreateCachedSegOpr(size_t topo_start,
     }
     on_complete();
   };
-  opr_names.pop_back();
+  opr_names.pop_back(); // delete the last ','
```

Comment:
Nitpick: Lint issue - At least two spaces is best between code and comments
As a general suggestion - `make lint` and `make lint` catches Lint issues (recommended to run these before pushing PR / commits